index.js

```diff
@@ -108,6 +108,13 @@ function WebTorrent (opts) {
     self.proxyOpts.proxyTrackerConnections = self.proxyOpts.proxyTrackerConnections !== false
     self.proxyOpts.proxyPeerConnections = self.proxyOpts.proxyPeerConnections !== false
 
+    if (self.proxyOpts.proxyTrackerConnections && !self.tracker.proxyOpts) {
+      if (!self.tracker) {
```

Comment:
self.tracker is already set to an empty object above (see https://github.com/yciabaud/webtorrent/blob/405a084da5406a59acf168460d038b449d26a5a3/index.js#L86), no need for this here?